src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Build/ImplicitlyTriggeredDebugBuildManager.cs

```diff
@@ -13,14 +12,13 @@
 namespace Microsoft.VisualStudio.ProjectSystem.VS.Build
 {
     /// <summary>
-    /// Tracks implicitly builds triggered as part of F5/Shift+F5 debugging commands and
-    /// skips analyzer execution for these builds by invoking into <see cref="IImplicitlyTriggeredBuildManager"/>.
+    /// Tracks implicitly builds triggered as part of F5/Ctrl+F5 debug/launch commands and
+    /// updates the <see cref="IImplicitlyTriggeredBuildManager"/> appropriately.
```

Comment:
Note that IImplicitlyTriggeredBuildManager interface is also used by builds triggered as part of Run Tests/Debug Tests commands. Unit test library themselves track invoking the APIs on this interface, it doesn’t rely on this type to detect an implicit build as part of those commands.

Reply:
Yeah, I tracked down those other uses of `IImplicitlyTriggeredBuildManager` while I was trying to understand why the interface was defined in CPS. This works out well for me, because I will need to distinguish between implicit builds started as part of F5/Ctrl+F5, and those started as part of other operations.